VKappProject/VKappProject/Model/Session.swift

```diff
@@ -0,0 +1,20 @@
+// Session.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// ВК синглтон
+class Session {
+    // MARK: - Public Properties
+
+    static let shared = Session()
+
+    // MARK: - Private Initializator
+
+    private init() {}
+
+    // MARK: - Public Properties
```

Comment:
```suggestion
    // MARK: - Public Properties
```
Не по гайду. Перенеси выше инста